pkg/controllers/config/controller.go

```diff
@@ -167,5 +170,41 @@ func (f *featureHandler) onChange(key string, obj *corev1.ConfigMap) (*corev1.Co
 			return obj, err
 		}
 	}
+
+	if err := f.updateConfigMap(obj); err != nil {
+		return obj, err
+	}
 	return obj, f.rContext.Start(f.ctx)
 }
+
+func (f *featureHandler) updateConfigMap(obj *corev1.ConfigMap) error {
+	conf, err := config.FromConfigMap(obj)
+	if err != nil {
+		return err
+	}
+	if conf.Features == nil {
+		conf.Features = map[string]config.FeatureConfig{}
+	}
+
+	state := f.featureState
+
+	t := true
+	for _, feature := range features.GetFeatures() {
+		f := conf.Features[feature.Name()]
+		if state.Has(feature.Name()) {
+			f.Enabled = &t
```

Comment:
Trying to think if there is a case where using same address for all the true's will flip all at once. Think its ok, since t never changes and gets re-instantiated every call. I like this a lot better than the slice method.